gatsby-config.js

```diff
@@ -57,10 +57,6 @@ products.forEach(name => {
   productIcons[name] = require(getProduct(name)).pathD
 })
 
-if (docsConfig.productLogoPathD && docsConfig.productIconKey) {
-  return Error("Set either `productLogoPathD` or `productIconKey` in docs-config.js, not both")
-}
-
 if (docsConfig.productIconKey) {
```

Comment:
Consider updating this line to something like the following:

```js
if (docsConfig.productIconKey && !docsConfig.productLogoPathD) {
```

Rationale: For example, Bots is using `productLogoPathD` for its icon, but you'll be adding a `productIconKey` entry in cloudflare/cloudflare-docs#2202:
https://github.com/cloudflare/cloudflare-docs/pull/2202/files#diff-29a8dd940efee313c021d705bd5fa7c376f9aec897226eaad2f101b4d96d6b76

If you don't change this line, the icon defined using `productIconKey` will override the icon defined by `productLogoPathD`, and the behavior will change.

Reply:
changed! to support this behavior thanks